message TestMessage{
	required int32 counter =6 [default = 0];
}
